doc/content/getting-started/aws/ecs/troubleshooting/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting AWS ECS Deployment"
+description: ""
+weight: 7
+---
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/2714
+Move to generic getting started guide once ready.
+-->
```

Comment:
That issue is closed already. What is this for?

Reply:
It was our mistake. Please ignore this, and we have removed it now.